capstan.go

```diff
@@ -423,6 +423,7 @@ func main() {
 						cli.BoolFlag{Name: "pull-missing, p", Usage: "attempt to pull packages missing from a local repository"},
 						cli.StringFlag{Name: "boot", Usage: "specify default config_set name to boot unikernel with"},
 						cli.StringSliceFlag{Name: "env", Value: new(cli.StringSlice), Usage: "specify value of environment variable e.g. PORT=8000 (repeatable)"},
+						cli.StringFlag{Name:"remote-guest-name",Usage:"specify hostname of remote guest"},
 					},
 					Action: func(c *cli.Context) error {
 						if len(c.Args()) != 1 {
```

Comment:
This check shoul be updated so that when `--remote` is given, the argument `image-name` is no longer needed.